patches/chrome-browser-ui-webui-settings-md_settings_ui.cc.patch

```diff
@@ -10,3 +10,12 @@ index f8e4ae44148aa01579097d4ef373025b66dde788..dde5c9c64bdeb0091e7b7780e295b5d4
  }
  
  MdSettingsUI::MdSettingsUI(content::WebUI* web_ui)
+@@ -155,7 +156,7 @@ MdSettingsUI::MdSettingsUI(content::WebUI* web_ui)
+ 
+   AddSettingsPageUIHandler(
+       std::make_unique<MediaDevicesSelectionHandler>(profile));
+-#if defined(GOOGLE_CHROME_BUILD) && !defined(OS_CHROMEOS)
++#if (defined(GOOGLE_CHROME_BUILD) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))) && !defined(OS_CHROMEOS)
```

Comment:
we can add this by using the preprocessor method to subclass `MdSettingsUI` and override it in `ChromeWebUIControllerFactory`